masterlist.yaml

```
@@ -832,6 +832,21 @@ plugins:
       - 'Walkers of Morrowind.esp'
 
 ###### Gameplay - Quests ######
+  - name: 'Guild of Vampire Hunters.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/45832/' ]
+    after:
+      - 'Beautiful cities of Morrowind.ESP'
+      - 'NX9_StuddedArmor.esp'
+  - name: 'LuceEdit - Guild of Vampire Hunters.esp'
```

Not from the same mod, so they shouldn't be placed together.

```suggestion

  - name: 'LuceEdit - Guild of Vampire Hunters.esp'
```